engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterPluginAppLifeCycleDelegate_internal.h

```diff
@@ -19,6 +19,22 @@
  */
 - (BOOL)hasPluginThatRespondsToSelector:(SEL)selector;
 
+/**
+ * Forwards the `application:didFinishLaunchingWithOptions:` lifecycle event to plugins if they have
+ * not received it yet. This compensates for the UIScene migration, which causes storyboards (and
+ * thus the plugin registration via `FlutterImplicitEngineDelegate`) to be instantiated after the
+ * application launch events.
+ */
+- (void)applicationDidFinishLaunchingSceneFallback:(UIApplication*)application;
```

Comment:
It's kind of gross, but technically this should be called something like `sceneFallbackDidFinishLaunchingApplication:`, since the "Application" part should be right before the `:`. (Same below.)